issueshark/backends/jirabackend.py

```diff
@@ -241,41 +229,42 @@ def _store_comments(self, jira_issue, mongo_issue_id):
         for comment in jira_issue.fields.comment.comments:
             logger.debug('Processing comment: %s' % comment)
             created_at = dateutil.parser.parse(comment.created)
-            try:
-                mongo_comment = IssueComment.objects(external_id=comment.id, issue_id=mongo_issue_id).get()
-                logger.debug('Comment already in database, id: %s' % mongo_comment.id)
-                continue
-            except DoesNotExist:
-                mongo_comment = IssueComment(
-                    external_id=comment.id,
-                    issue_id=mongo_issue_id,
-                    created_at=created_at,
-                    author_id=self._get_people(comment.author.name, self._get_user_email(comment.author),
-                                               comment.author.displayName),
-                    comment=comment.body,
-                )
-                logger.debug('Resulting comment: %s' % mongo_comment)
-                comments_to_insert.append(mongo_comment)
+            mongo_comment = None
+            if mongo_issue:
+                try:
+                    mongo_comment = IssueComment.objects(external_id=comment.id, issue_id=mongo_issue.id).get()
```

Comment:
No, because Issue id is unique for every time we update the data. For example, if we have updated data we will create a new issue id therefore there is no problem here.